### DISCUSSION TITLE
Uninstall VMExpress?

### DISCUSSION THREAD

**Discussion by andyszymas** [amazon-connect/amazon-connect-salesforce-scv#161] [Q&A]
I'm in the process of upgrading several instances to use the new voicemail solution for SCV; are there any directions for what needs to be done to remove VMExpress?

**Comment by dougjaso** [comment]
https://github.com/amazon-connect/amazon-connect-salesforce-scv/blob/master/Solutions/VMX2-VoicemailExpress/Docs/vmx_uninstall.md
